parm/use_cases/model_applications/s2s/GridStat_SeriesAnalysis_fcstNMME_obsCPC_seasonal_forecast.conf

```diff
@@ -20,7 +20,7 @@ INIT_END = 201007
 INIT_INCREMENT = 1Y
```

Comment:
Are the changes in this file really related to the new gen_vx_mask wrapper?  It wasn't immediately obvious why it would be, but I could easily be not understanding.

Reply:
The initial comment mentions that some other changes that were unrelated accidentally made it into this pull request. This is one of them. Sorry about that.